scripts/wallet.js

```diff
@@ -811,7 +810,7 @@ export class Wallet {
     subscribeToNetworkEvents() {
         getEventEmitter().on('new-block', async (block) => {
             if (this.#isSynced) {
-                stakingDashboard.update(0);
+                await getNetwork().getLatestTxs(this);
```

Comment:
getLatestTxs must not be called anymore after first sync
`await this.getLatestBlocks(block);` will handle both